app/views/waste_calendar/create.html.erb

```diff
@@ -0,0 +1,63 @@
+<div class="d-sm-flex align-items-center justify-content-between">
+  <h1 class="h3 mb-0 text-gray-800">Schritt 2: Daten für Import zuweisen</h1>
+</div>
+
+<p>Die Spalten aus beiden CSV-Quellen werden nun den Feldern aus dem Abfallkalender zugewiesen.</p>
+
+<div class="row">
+  <div class="col-lg-8">
+    <%= form_with( url: "/waste_calendar", method: :post, local: true) do |f| %>
+      <%= hidden_field_tag :address_data, @address_data %>
+      <%= hidden_field_tag :tour_data, @tour_data %>
+      <div class="row">
+        <div class="col">
+          <h2 class="d-sm-flex align-items-center justify-content-between my-4">
+            Adressdaten
+          </h2>
+          <div class="form-group">
+            <label for="description">Straße:</label>
+            <div> <%= select_tag("address[street]", options_for_select(@address_data_selects), include_blank: true, class: "form-control" ) %></div>
+          </div>
+          <div class="form-group">
+            <label for="description">PLZ:</label>
+            <div> <%= select_tag("address[zip]", options_for_select(@address_data_selects), include_blank: true, class: "form-control" ) %></div>
+          </div>
+          <div class="form-group">
+            <label for="description">Stadt:</label>
+            <div> <%= select_tag("address[city]", options_for_select(@address_data_selects), include_blank: true, class: "form-control" ) %></div>
```

Comment:
wenn nichts bei den selects im Schritt 2: ausgewählt wird, dann kommt trotzdem import startet aber geht natürlich nicht. werde ich noch abfangen!

Reply:
done in f61be1c4f4fe810d46c149ff876c048e8952bf32

Reply:
wird nochmal angepasst, da es mehr waste types geben kann als auswahl nötig sind.
also prüfung auf mindestens eine zuordnung machen.

Reply:
ansonsten neue flash meldung:

![Bildschirmfoto 2021-02-25 um 14 55 52](https://user-images.githubusercontent.com/1942953/109163638-d4362b00-7779-11eb-911c-5bf4275f333e.png)